README.md

```diff
@@ -103,7 +103,7 @@ To install the MapboxNavigation framework using [Carthage](https://github.com/Ca
    # Latest stable release
    github "mapbox/mapbox-navigation-ios" ~> 2.5
```

Comment:
I think this can be updated to `2.5.1` already.

Reply:
Because `2.5.1` release isn't on CocoaPods. So should we keep them synced with the `2.5.0` or just update the Carthage and the SPM?

Reply:
That's a miss from my side on `v2.5.1`. I've just pushed it and it should be available shortly.
Considering updating the guides, it's not there because `release-2.5` isn't merged into `main` after 2.5.1. release. We could run `update-version.sh` with `2.5.1` just to correctly update all the guides. cc: @S2Ler